farmdata2_modules/fd2_tabs/resources/FarmOSAPI.js

```diff
@@ -42,9 +42,57 @@ function getAllPages(url, arr) {
     })
 }
 
+function getIDToCropMap(){
+    return getMap('/farm_asset.json?type=planting', 'id', 'name')
```

Comment:
This should use the farm_crops bundle from the taxonmy_term.json endpoint rather than plantings.  All of the crops that can be used will be defined in the farm_crops vocabulary.